src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costPerDay = 40;
+  const threeDaysDiscount = 20;
+  const sevenDayDiscount = 50;
+
+  if (days < 3) {
+    return days * costPerDay;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days < 7) {
```
ти вже перевірила що воно більше 3 в попередньому if, тому повторно це робити не треба